src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SORT;
+import static seedu.address.model.Model.COMPARATOR_SORT_BY_NAME;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.SortOption;
+import seedu.address.model.Model;
+
+/**
+ * Sorts the person list based on the specified option.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the person list.\n"
+            + "If no sort option is provided, the list is restored to its original order (i.e., order of addition).\n"
+            + "Parameters: " + "[" + PREFIX_SORT + "SORT_OPTION]\n"
+            + "Example: " + COMMAND_WORD + " s/name";
+
+    public static final String MESSAGE_DEFAULT_SUCCESS = "Sorted by default order";
+    public static final String MESSAGE_SORT_SUCCESS = "Sorted by %s";
+    public static final String MESSAGE_UNSUPPORTED_SORT_OPTION = "Unsupported sort option: %s";
+
+    private final SortOption sortOption;
+
+    public SortCommand() {
+        this.sortOption = null;
+    }
+
+    public SortCommand(SortOption sortOption) {
+        this.sortOption = sortOption;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (sortOption == null) {
+            model.clearPersonSort();
+            return new CommandResult(MESSAGE_DEFAULT_SUCCESS);
+        }
+
+        switch (sortOption.toString()) {
+        case SortOption.SORT_NAME:
```

Comment:
for this are we able to import at the top, then simply write SORT_NAME

Reply:
Sure that sounds great